### PR TITLE
remove blist and dlist

### PR DESCRIPTION
**Pull request by Gankro**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -37,6 +36,9 @@ optional = true
 [dependencies.traverse]
 optional = true
 
+[dependencies.blist]
```
Comment (apasel422) [inline Cargo.toml]: Is this still necessary?

Comment (Gankro) [inline Cargo.toml]: Oh whoops, I mixed something up.

**Comment by Gankro** [comment]
redundant deps removed

**Comment by apasel422** [comment]
Have to remove this line from `.travis.yml`:
```yaml
 - cargo test --no-default-features --features "blist"
```

**Comment by Gankro** [comment]
Whoops. Done.

**Pull request merged**